web-common/src/features/dashboards/time-series/MeasureChart.svelte

```diff
@@ -69,10 +69,10 @@
   $: mouseoverFormat = createMeasureValueFormatter<null | undefined>(measure);
   $: numberKind = numberKindForMeasure(measure);
 
-  export let tweenProps = { duration: 400, easing: cubicOut };
-
+  const tweenProps = { duration: 400, easing: cubicOut };
   const xScale = getContext(contexts.scale("x")) as ScaleStore;
 
+  let hovered: boolean | undefined = false;
```

Comment:
Nit: given `hovered` is initialized to `false`, it doesn't look like it'll ever be `undefined`?